src/test/regress/multi_schedule

```diff
@@ -90,6 +90,7 @@ test: multi_basic_queries multi_complex_expressions multi_subquery multi_subquer
 test: multi_subquery_complex_reference_clause multi_subquery_window_functions multi_view multi_sql_function multi_prepare_sql
 test: sql_procedure multi_function_in_join row_types materialized_view
 test: multi_subquery_in_where_reference_clause full_join adaptive_executor propagate_set_commands
+test: rollback_to_savepoint
```

Comment:
can we run it in parallel with other tests where parallelism is low?

Reply:
For some weird reason when running the test in parallel I get distributed deadlocks, instead of timeouts: https://app.circleci.com/pipelines/github/citusdata/citus/9575/workflows/c434721e-d812-45ef-b830-1b86ee3f5d78/jobs/136125/steps

So I'll keep the test as is.